go.mod

```diff
@@ -58,26 +57,31 @@ require (
 	google.golang.org/protobuf v1.27.1
 )
 
+require (
```

Comment:
is it really canonical in golang 1.17 to have 3 require blocks?

Reply:
No need to... I've unified all the packages under one require block